R/utils.R

```diff
@@ -23,6 +23,40 @@ check_tibble <- function() {
   }
 }
 
+check_tidyselect <- function(){
+  if (!is_installed("tidyselect")) {
+    abort("Using tidyselect in `map_at()` requires tidyselect")
+  }
+}
+
+#' Select names
+#'
+#' This helper function is intended to provide a semantic equivalent to the one from `dplyr::mutate_at()`. It can be used in the `*_at` functions of `purrr` like `map_at()`. Note that the `tidyselect` package has to be installed.
```

Comment:
Could you hard-reflow the paragraph at 80 characters please. Same for the `@param` section.